NUSense/Core/Src/test_hw.hpp

```diff
@@ -131,5 +131,4 @@ namespace test_hw {
 #endif
 
 }  // namespace test_hw
```

Comment:
```suggestion
}  // namespace test_hw

```
Just quickly change this for me.